packages/api-openfin/message.js

```diff
@@ -0,0 +1,28 @@
+if (!window.ssf) {
+  window.ssf = {};
+}
+
+class MessageService {
+  // Window ID should be in the form 'application-uuid:window-name'
+  static send(windowId, topic, message) {
+    const splitId = windowId.split(':');
```

Comment:
Could do something like this here, if OpenFin supports this destructuring syntax:

```js
const  [appId, windowName] = windowId.split(':')
```